packages/core/src/checks/is-firehose-webhook.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (method) => {
+  return (
+    method.startsWith('firehoseWebhooks.') &&
```

Comment:
Should `firehoseWebhooks.whatever.performSubscriptionKeyList` be a valid name? It doesn't seem like it, so this could just check if the method is exactly `firehoseWebhooks.performSubscriptionKeyList`

Reply:
haha oops. I was just flowing with the style of the other checks 😅